vibe_code_apps_20/wattlab/studio/pages/fuel_dashboard.py

```
@@ -131,13 +131,23 @@ def render(*, campus: Campus | None = None) -> None:
+        "registry (kBtu/ft²-yr). Twin page adds EnergyPlus model EUI beside these bands."
+    )
     rows = []
     for b in summary["buildings"]:
         cmp = compare_eui(b["site_eui_kbtu_ft2"], b["property_type"])
         rows.append({**b, **{f"peer_{k}": cmp[k] for k in ("p20", "p50", "p80", "band", "vs_median_pct")}})
     dfb = pd.DataFrame(rows)
+    b0 = rows[0]
+    p1, p2, p3, p4 = st.columns(4)
+    p1.metric("Bill site EUI", f"{b0['site_eui_kbtu_ft2']} kBtu/ft²")
+    p2.metric("Peer typical (p50)", f"{b0['peer_p50']} kBtu/ft²")
+    p3.metric("Peer p20–p80", f"{b0['peer_p20']} – {b0['peer_p80']}")
```

Ruff (0.15.21)

[warning] String contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF001)